include/deal.II/matrix_free/cuda_matrix_free.templates.h

```diff
@@ -746,53 +704,44 @@ namespace CUDAWrappers
     dofs_per_cell     = fe.n_dofs_per_cell();
     q_points_per_cell = Utilities::fixed_power<dim>(n_q_points_1d);
 
-    const ::dealii::internal::MatrixFreeFunctions::ShapeInfo<Number> shape_info(
-      quad, fe);
+    ::dealii::internal::MatrixFreeFunctions::ShapeInfo<Number> shape_info(quad,
+                                                                          fe);
 
     unsigned int size_shape_values = n_dofs_1d * n_q_points_1d;
 
-    FE_DGQArbitraryNodes<1> fe_quad_co(quad);
-    const ::dealii::internal::MatrixFreeFunctions::ShapeInfo<Number>
-      shape_info_co(quad, fe_quad_co);
+    FE_DGQArbitraryNodes<1>                                    fe_quad_co(quad);
+    ::dealii::internal::MatrixFreeFunctions::ShapeInfo<Number> shape_info_co(
+      quad, fe_quad_co);
```

Comment:
I think we should remove these three lines, they are no longer necessary since #8854, see also the suggestions below.